src/task.c

```diff
@@ -474,6 +493,7 @@ task_group_deinit(task_group *group)
 static platform_status
 task_group_init(task_group  *group,
                 task_system *ts,
+                const char  *task_type, // Name of tasks in this group
```

Comment:
The only thing this is used for is to do the printf inside this function.  Move the printf to the caller and delete this parameter.

Reply:
Move print call on L528 to caller, and do not add this new param.

Reply:
Reworked to remove the new param `task_type` to this function.

I added a print msg to the caller when # of bg-threads of a specific type is non-zero.

But from previous discussions, the leaning of the team seems to be not to have too many extraneous messages. I am ok with having this message come out, as it helps me see this creation happening in other integrations, where I troll through the logfile.

If anyone thinks this is still too much and insist that this be removed, I can do that.